### ISSUE TITLE
Missing installation steps for Linux

### ISSUE THREAD

**Issue by Schlaubergen** [royshil/obs-backgroundremoval#91]
Hello,

is there a complete installation instruction for a linux newbie like me? I am using Linux MX, based on Debian.

The [first three steps](https://github.com/royshil/obs-backgroundremoval#ubuntu) seemed to work well (sudo / wget / sudo), except:

At the very first time I missed the "sudo" in the first line, therefor I accidently installed it as root and not as user. I don't know if a deinstallation is mandatory before a new installation. But the installation tries after it seemed to work fine, except:

I couldn't install it with "language-pack-en" (I also tried "language-pack-de"):
`E: Paket language-pack-en kann nicht gefunden werden. 
`
**Is this error message relevant?**

**How and when do I have to download the GitHub project?**

When I manually download the last sourcecode version and unzip it to the build folder, it says:
```
Me@MX:~/build
$ cmake .. && cmake --build . && sudo cmake --install .
CMake Error: The source directory "/home/me" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI. 
```

When changing to a subfolder, the result is:
```
Me@MX:~/build
$ cd installer
Me@MX:~/build/installer
$ cmake .. && cmake --build . && sudo cmake --install .
-- Found Git: /usr/bin/git (found version "2.30.2")
CMake Error at external/MunkeiVersionFromGit.cmake:65 (message):
  [MunkeiVersionFromGit] Failed to execute Git: fatal: Kein Git-Repository
  (oder irgendein Elternverzeichnis bis zum Einhängepunkt /)
  Stoppe bei Dateisystemgrenze (GIT_DISCOVERY_ACROSS_FILESYSTEM nicht
  gesetzt).
Call Stack (most recent call first):
  CMakeLists.txt:4 (version_from_git)
-- Configuring incomplete, errors occurred! 
```

When copying the TXT file into my user folder, it says:
```
Me@MX:~/build
$ cmake .. && cmake --build . && sudo cmake --install .
CMake Error at CMakeLists.txt:3 (include):
  include could not find load file:
    external/MunkeiVersionFromGit.cmake
CMake Error at CMakeLists.txt:4 (version_from_git):
  Unknown CMake command "version_from_git".
-- Configuring incomplete, errors occurred!
```

**Comment by Schlaubergen** [comment]
Ok, with a new research I finally found the solution in the [MX Linux forum](https://forum.mxlinux.org/viewtopic.php?t=41483).

Right after the first three steps (sudo / wget / sudo) this is needed:
```
git clone https://github.com/royshil/obs-backgroundremoval
cd obs-backgroundremoval
```
At first I was disillusioned, but then I saw that there were also steps to do after the two lines "mkdir" and "cmake". Until then I thought that everything below was only relevant for Nvidia users with CUBA support.
